src/main/proto/netflix/titus/validator/validator.proto

```diff
@@ -251,3 +257,18 @@ message ImageValidationResponse {
 
   CacheMetadata cacheMetadata = 3;
 }
+
+message PlatformSidecarValidationRequest {
+  netflix.titus.PlatformSidecar platformSidecar = 1;
```

Comment:
I think it will be more efficient (but slightly more complex) to do a batch query here (add `repeated`) so that the api/spinnaker/whatever doesn't have to make serial calls for the entire array that a job has.

In fact, I'm not super sure why this API isn't just 1 object: the job? idk

Reply:
The issue I ran into when considering that is how would the REST side of this look? It's simple to take query parameters corresponding to the fields inside `PlatformSidecar`, but if we accept multiple `PlatformSidecar`s, it gets awkward to do with query parameters. Would it be reasonable to limit the REST implementation to a single sidecar reference at a time?